SemanticUI/Modules/Dropdown/Select.elm

```diff
@@ -0,0 +1,327 @@
+
+-}
+type Variation msg
+    = Ordinary
+    | Button (Button.Config msg)
+    | Inline
+    | Selection { compact : Bool }
+
+
+variationClassList : Variation msg -> List ( String, Bool )
+variationClassList variation =
+    case variation of
+        Ordinary ->
+            []
+
+        Button _ ->
+            []
+
+        Inline ->
+            [ ( "inline", True ) ]
+
+        Selection { compact } ->
+            [ ( "compact", compact ), ( "selection", True ) ]
+
+
+{-| Most general configuration that applies any `Select`.
+
+It is recommended that you use `inline`, `button`, or `select` to construct this record.
+
+-}
+type alias Config msg option =
+    { variation : Variation msg
+    , drawerState : Drawer.State
+    , identifier : String
+    , onSelect : option -> msg
+    , onToggle : Drawer.State -> msg
+    , attributes : List (Attribute msg)
+    , toggleEvent : Toggle.Event
+    , disabled : Bool
+    , optionAttributes : option -> List (Attribute msg)
+    , labels : List (Html msg)
+    , caret : Bool
+    , fluid : Bool
+    , scrolling : Bool
+    }
+
+
+{-| A type that represents the dropdown select.
+
+Use `inline`, `button`, or `select` to construct this record and `toHtml` or `toCustomHtml` to render it.
+
+-}
+type Select msg option
+    = Select (Config msg option)
+
+
+{-| Any other custom `Attribute`s to add to the select. Custom attributes
+will be added before `elm-semantic-ui` attributes.
+
+Identical to `Dropdown.attributes`
+
+-}
+attributes : List (Attribute msg) -> Select msg option -> Select msg option
+attributes a (Select config) =
+    Select { config | attributes = a }
+
+
+{-| Set `toggleEvent` on any `Select`
+
+Identical to `Dropdown.toggleEvent`
+
+-}
+toggleEvent : Toggle.Event -> Select msg option -> Select msg option
+toggleEvent a (Select config) =
+    Select { config | toggleEvent = a }
+
+
+{-| Set `disabled` on any `Select`.
+
+Identical to `Dropdown.disabled`
+
+-}
+disabled : Bool -> Select msg option -> Select msg option
+disabled a (Select config) =
+    Select { config | disabled = a }
+
+
+{-| Set `caret` on a `Select`.
+
+Identical to `Dropdown.caret`
+
+-}
+caret : Bool -> Select msg option -> Select msg option
+caret a (Select config) =
+    Select { config | caret = a }
+
+
+{-| Set `fluid on a`Select\`.
+
+Identical to `Dropdown.fluid`
+
+-}
+fluid : Bool -> Select msg option -> Select msg option
+fluid a (Select config) =
+    Select { config | fluid = a }
+
+
+{-| Set `scrolling` on a `Select`.
+
+Identical to `Dropdown.scrolling`
+
+-}
+scrolling : Bool -> Select msg option -> Select msg option
+scrolling a (Select config) =
+    Select { config | scrolling = a }
+
+
+{-| Everything needed to build the `Html msg` representation of a particular select dropdown.
+
+  - toDropdown - converts a `<div>` or an `<a>` element into a SemanticUI dropdown
+  - toOption - takes the option value and converts a `<div>` or an `<a>` element into a selectable option item
```

Comment:
```suggestion
  - toOption - takes a dropdown option value and renders it as a HTML element that is a selectable option item
```

Reply:
This is also a case where only `<div>` and `<a>` really makes sense for the `item` class. Other elements won't layout consistently.